app/views/simple_smart_answers/flow.html.erb

```diff
@@ -1,5 +1,14 @@
-<% content_for :extra_headers do %>
-  <meta name="robots" content="noindex" />
+<% 
+  ga4_attributes = {
+    event_name: "information_click",
+    type: "simple smart answer",
+    section: @flow_state.current_node.title,
+    action: "information_click",
+    tool_name: publication.title,
+  }.to_json
+
+  content_for :extra_headers do %>
+    <meta name="robots" content="noindex" />
```

Comment:
Is the indentation for this block correct? Just as the `<% end %>` looks off from the `content_for :extra_headers do`

Reply:
Good spot, have updated 👍